packages/core/package.json

```diff
@@ -13,10 +13,11 @@
   },
   "devDependencies": {
     "@ethereum-waffle/provider": "^3.2.2",
+    "@testing-library/react-hooks": "^5.0.3",
     "@types/chai": "^4.2.14",
     "@types/chai-as-promised": "^7.1.3",
-    "@types/mocha": "^8.2.0",
     "@types/lodash": "^4.14.170",
+    "@types/mocha": "^8.2.0",
```

Comment:
since you aren't changing anything in packages/core you can restore this package.json to master